pkg/testmachinery/metadata/types.go

```diff
@@ -26,9 +26,10 @@ type Metadata struct {
 	Region            string `json:"region,omitempty"`
 
 	// todo: schrodit - add support to better persist multiple worker pools with multiple oss, versions and zones
-	OperatingSystem        string `json:"operating_system,omitempty"`
-	OperatingSystemVersion string `json:"operating_system_version,omitempty"`
-	Zone                   string `json:"zone,omitempty"`
+	OperatingSystem           string `json:"operating_system,omitempty"`
+	OperatingSystemVersion    string `json:"operating_system_version,omitempty"`
+	Zone                      string `json:"zone,omitempty"`
+	AllowPrivilegedContainers *bool  `json:"allowPrivilegedContainers,omitempty"`
```

Comment:
shouldn't it be snake case `allow_privileged_containers ` as operating system? 
I can't really what we agreed on for the elasticsearch metadata structure

Reply:
Guess it doesn't matter for elasticsearch, but of course it should follow what we already, so snake case is the way, thanks for spotting.